com.hostbuf.FinalShell.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id": "com.hostbuf.FinalShell",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "FinalShell",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=x11",
```

Comment:
why do you need both? EIther it supports wayland and it's fallback-x11, or it doesn't and then there is no socket for it.